<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVista2D</class>
 <widget class="QMainWindow" name="CVista2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vista 2D</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsViewLocal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewGlobal">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderEscala">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMostrarEsferas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Esferas</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMostrarCajas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Cajas</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMostrarPersonas">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Personas</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxMostrarRobots">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar Robots</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServidor">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemoto">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remoto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
